test/examples.jl

```diff
@@ -3,14 +3,12 @@ using ModelingToolkit, EasyModelAnalysis
 Dₜ = Differential(t)
 @variables S(t)=0.9 Iₐ(t)=0.05 Iₛ(t)=0.01 Rₐ(t)=0.2 Rₛ(t)=0.1 D(t)=0.01
 @parameters α=0.6 βₐ=0.143 βₛ=0.055 ρ=0.003 μₙ=0.007 μₘ=0.011 θ=0.1 ωₛ=0.14
-eqs = [
-       Dₜ(S) ~ μₙ*S - μₘ*S - θ*α*S*Iₛ - (1-θ)*α*S*Iₐ + ρ*(Rₐ + Rₛ)
-       Dₜ(Iₐ) ~ (1 - θ)*α*S*Iₐ - βₐ*Iₐ
-       Dₜ(Iₛ) ~ θ*α*S*Iₛ - βₛ*Iₛ
-       Dₜ(Rₐ) ~ βₐ*Iₐ - ρ*Rₐ
-       Dₜ(Rₛ) ~ (1 - ωₛ)*βₛ*Iₛ - ρ*Rₛ
-       Dₜ(D) ~ ωₛ*βₛ*Iₛ
-      ]
+eqs = [Dₜ(S) ~ μₙ * S - μₘ * S - θ * α * S * Iₛ - (1 - θ) * α * S * Iₐ + ρ * (Rₐ + Rₛ)
+       Dₜ(Iₐ) ~ (1 - θ) * α * S * Iₐ - βₐ * Iₐ
+       Dₜ(Iₛ) ~ θ * α * S * Iₛ - βₛ * Iₛ
+       Dₜ(Rₐ) ~ βₐ * Iₐ - ρ * Rₐ
+       Dₜ(Rₛ) ~ (1 - ωₛ) * βₛ * Iₛ - ρ * Rₛ
+       Dₜ(D) ~ ωₛ * βₛ * Iₛ]
 @named asir = ODESystem(eqs)
```

Comment:
? Is your formatter setup incorrectly?

Reply:
The only thing I have different is that it should format markdown in docstrings. No idea why that breaks it.